client/src/components/ChannelCard.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import styled from "@emotion/styled";
+import PropTypes from "prop-types";
+
+const Card = styled.div`
+  display: grid;
+  grid-template-columns: repeat(4, 25%);
+  grid-template-rows: repeat(4, 25%);
+  border-radius: 10%;
+  width: 170px;
+  height: 170px;
+  background-color: #ffffff;
+`;
+
+const Circle = styled.div`
+  grid-column: 2/4;
+  grid-row: 2/4;
+  background-color: #25386426;
+  background-image: url(${({ imgSrc }) => imgSrc});
+  border-radius: 100%;
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 5em;
+`;
+
+const ChannelName = styled.p`
+  text-align: center;
+  /* grid-column: 1/5;
```

Comment:
Do you still need this code? It might be better just to remove the commented out code if it's not necessary.

Reply:
You´re right! I just removed this code.